styles/styles.css

```diff
@@ -0,0 +1,87 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap");
+
+:root {
+  --white: #ffffff;
+}
+
+body {
+  margin: 0px;
+  font-family: "Inter", sans-serif;
+}
+
+p {
+  margin: 0px;
```

Comment:
Instead of 0px you can give the value without the unit too. Simply margin: 0;

Reply:
thankyou for this review sure I will update these things.